apps/dashboard/src/components/billing/plan-switcher.tsx

```diff
@@ -1,8 +1,9 @@
 import { Tabs, TabsList, TabsTrigger } from '@/components/primitives/tabs';
+import { StripeBillingIntervalEnum } from '@novu/shared';
```

Comment:
Ideally, most billing-related codes should live outside the public mono repo. We will discuss this offline.